### ISSUE TITLE
Multiple RegExp

### ISSUE THREAD

**Issue by ayman-gbt** [micazi/rich_text_controller#48]
rich_text_controller: ^2.0.1

When I use both to match texts starting with # or @ nothing happens but if I use only one it works 

targetMatches: [
        MatchTargetItem(
          regex: RegExp(r"#\w+"),
          style: TextStyle(
            color: primaryColor,
            fontStyle: FontStyle.italic,
          ),
          allowInlineMatching: true,
        ),
        MatchTargetItem(
          regex: RegExp(r"@\w+"),
          style: TextStyle(
            color: primaryColor,
            fontStyle: FontStyle.italic,
          ),
          allowInlineMatching: true,
        ),
      ],

**Comment by viJay-beep-code** [comment]
This issue I see was fixed in the code with commit (https://github.com/micazi/rich_text_controller/commit/d24096442626ff8abdbac3b385efe88af511f215) but not released on pub dev.
you can add ref to git repo for now in pubspec file -

```
  rich_text_controller:
    git:
      url: https://github.com/micazi/rich_text_controller.git
```

**Comment by ayman-gbt** [comment]
> This issue I see was fixed in the code with commit ([d240964](https://github.com/micazi/rich_text_controller/commit/d24096442626ff8abdbac3b385efe88af511f215)) but not released on pub dev. you can add ref to git repo for now in pubspec file -
> 
> ```
>   rich_text_controller:
>     git:
>       url: https://github.com/micazi/rich_text_controller.git
> ```

Working amazing .Thanks.
